docusaurus.config.js

```diff
@@ -7,7 +7,7 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 /** @type {import('@docusaurus/types').Config} */
 const config = {
   title: 'Knockout City Launcher',
-  tagline: 'Brawlin doesnt have to end. Join the community.',
```

Comment:
Keep it as "Brawlin' doesn't have to end. Join the community!" cause that resembles the original tone of knockout city